### ISSUE TITLE
Lot of open PRs

### ISSUE THREAD

**Issue by santiagodoldan** [Reklino/angular-resizable#71]
@Reklino I was wondering if you want another dev to continue updating the gem, I can help with that, let me know and I can create a fork and start merging those PRs and adding new features

**Comment by abkieling** [comment]
This is a great project. Let's keep it active. I'd prefer to contribute to the project instead of forking it.

**Comment by stanleyxu2005** [comment]
Yes. This repo looks not to be maintained actively. I hope we are able to create a new forked project without violating any license.
